_episodes/14-checkout.md

```diff
@@ -0,0 +1,313 @@
+---
```

Comment:
Note: the diff for checkout.md is not viewable here due to episode renumbering. In addition, much content has been transferred from the other checkout page in Extras. However, diff against the prior checkout.md page can be [viewed here. ](https://github.com/data-lessons/instructor-training/pull/43/files)